i18n/locales/en.ts

```diff
@@ -2806,33 +2806,34 @@ Stack Hint: To spend before the secret is revealed, Vanderpoole uses his signatu
-        'Before I sign this, I need a guarantee that your full refund transaction can be revoked.',
-      paragraph_four: `Once you actually pay me for the beer, you shouldn't be able to broadcast that. And if you DO try to broadcast the revoked transaction, I get to keep ALL 100000 satoshis!"`,
+        'Before I sign this, I need a guarantee that this refund transaction can be revoked.',
+      paragraph_four: `Once you actually pay me for the beer, you shouldn't be able to broadcast this refund transaction. And if you DO try to broadcast it after it has been revoked, I get to keep <span className="italic">all</span> 100,000 satoshis!"`,
     },
     updating_the_state_three: {
       title: 'The revocation',
       nav_title: 'The revocation',
       heading_one: 'The revocation',
-      paragraph_one: `You can make the output to yourself revocable by Laszlo by adding an extra condition to the 100000 satoshi output. The logic branch should allow Laszlo to spend the output with his own key AND a new private key that you generate. To revoke the transaction, you will just give Laszlo the new private key. It's a very unusual thing to do! But since it puts 100000 satoshis at stake, it proves to Laszlo you will NOT broadcast it after revoking it.`,
-      paragraph_two: `You generate a new key pair: a private key revocation_you_1 and PUBKEY(REVOCATION_YOU_1). You will generate a new key pair like this every time you want to update the state of the payment channel`,
+      paragraph_one: `You can make the 100,000 satoshi output to yourself revocable by Laszlo if you add an extra condition to it. The logic branch should allow Laszlo to spend the output with his own key AND a new private key that you generate. To revoke the transaction, you give Laszlo the new private key. It's a very unusual thing to do, but it puts 100,000 satoshis at stake, and allows Laszlo to sweep the funds if you try to broadcast the transaction after revoking it.`,
+      paragraph_two: `You generate a new key pair: a private key <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base">REVOCATION_YOU_1</span> and <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base">PUBKEY(REVOCATION_YOU_1)</span>. You will generate a new key pair like this every time you want to update the state of the payment channel.`,
+      heading_two: `Instructions`,
       revocation_list_one: `Add an IF condition to the script so Laszlo can spend the output ONLY if he ALSO has the revocation key (you will continue keeping the key secret until it is time to revoke!)`,
-      revocation_list_two: `Send it to Laszlo by clicking <span className="rounded-sm px-1.5 py-1 h-[28px] bg-[#0000004D] m-0.5 text-base">Send to Laszlo</span> so he can sign it`,
+      revocation_list_two: `Send it to Laszlo by clicking the <span className="rounded-sm px-1.5 py-1 h-[28px] bg-[#0000004D] m-0.5 text-base">Sign</span> button next to Laszlo's icon.`,
```

Comment:
@benalleng did I see somewhere that you updated the buttons? It's been a while since I looked at this PR (it was on my fork of the repo). I think I need to update this?

Reply:
yes, we transitioned it over to send to laszlo on all the pages where both you and laszlo need to sign.
https://github.com/saving-satoshi/saving-satoshi/blob/2446fa8f16e6dca601a99e35901484cd79bfe9dd/ui/lesson/TransactionsChallenge/index.tsx#L453C31-L457C42

Reply:
So the old copy i think can stay

Reply:
okie dokie, fixed it and rebased to resolve the conflicts. After this is merged, I have one more PR that's been sitting in my fork that I need to resolve conflicts on but that should be the last of my batch of updates from initial QA two weeks ago.